src/renderer/i18n/locales/it-IT.js

```diff
@@ -0,0 +1,964 @@
+    RESENT_NOTICE: 'La transazione {transactionId} è stata inviata di nuovo',
+    SAVE_OFFLINE: 'Salva transazione offline',
+    SECOND_PASSPHRASE: 'Seconda frase segreta',
+    SEND: 'Invia',
+    SEND_ALL: 'Invia Tutto',
+    SENDER: 'Mittente',
+    SINGLE: 'Singolo',
+    TIMESTAMP: 'Timestamp',
+    TRANSACTION: 'Transazione',
+    VENDOR_FIELD: 'Smartbridge',
+    WELL_CONFIRMED: 'Ben confermata',
+    WELL_CONFIRMED_COUNT: 'Ben confermata ({confirmations} conferme)'
+  },
+
+  SIGN_VERIFY: {
+    VERIFY_WALLET: 'Verifica il tuo portafoglio',
+    VERIFY_BY_SIGNING: 'Firma un messaggio per verificare di essere il proprietario del portafoglio',
+    SIGN: 'Firma',
+    VERIFY: 'Verifica',
+    TITLE_SIGN: 'Firma un messaggio',
+    TITLE_VERIFY: 'Verifica un messaggio',
+    MESSAGE: 'Messaggio',
+    ADDRESS: 'Indirizzo',
+    PUBLIC_KEY: 'Chiave pubblica',
+    SIGNATURE: 'Firma',
+    JSON_MESSAGE: 'Contenuto messaggio firmato',
+    FORMAT_FOOTER: 'Formato (JSON): { "publickey": "...", "signature": "...", "message": "..." }',
+    VERIFIED: 'Il messaggio è stato verificato con successo',
+    NOT_VERIFIED: 'Il messaggio NON è verificato',
+    CONFIRMATION: 'Conferma',
+    DELETE: 'Elimina messaggio',
+    FAILED_SIGN: 'Impossibile firmare messaggio',
+    FAILED_VERIFY: 'Impossibile verificare messaggio',
+    SUCCESSFULL_SIGN: 'Il tuo messaggio è stato firmato',
+    SUCCESSFULL_VERIFY: 'Il tuo messaggio è stato verificato'
+  },
+
+  SYNCHRONIZER: {
+    VOTE: 'votato per',
+    UNVOTE: 'voto rimosso',
+    NEW_SECOND_SIGNATURE: 'Nuova Seconda firma: {address}',
+    NEW_DELEGATE_REGISTRATION: 'Nuovo delegato: {address} registrato come {username}',
+    NEW_VOTE: 'Nuovo voto: {address} {voteUnvote} {publicKey}',
+    NEW_TRANSFER_SENT: 'Nuovo trasferimento: {amount} inviato da {sender} verso {recipient}',
+    NEW_TRANSFER_RECEIVED: 'Nuovo trsferimento: {amount} ricevuto da {sender} verso {recipient}'
+  },
+
+  TRANSACTION_FORM: {
+    ERROR: {
+      NOT_ENOUGH_BALANCE: 'Il saldo è troppo basso ({balance})'
+    }
+  },
+
+  TRANSACTION_DETAIL: {
+    SEND_AMOUNT: 'Invia {token}'
+  },
+
+  WALLET_HEADING: {
+    ACTIONS: {
+      WALLET_NAME: 'Nome portafoglio',
+      CONTACT_NAME: 'Nome contatto',
+      REGISTER_DELEGATE: 'Registra Delegato',
+      SECOND_PASSPHRASE: '2ª frase segreta',
+      DELETE_WALLET: 'Elimina portafoglio',
+      SHOW_PUBLIC_KEY: 'Mostra chiave pubblica',
+      SHOW_ADDRESS: 'Mostra indirizzo'
+    },
+    SECOND_PASSPHRASE_ENABLED: 'Seconda firma abilitata'
+  },
+
+  WALLET_SECOND_SIGNATURE: {
+    NEW: 'Genera nuova seconda frase segreta',
+    ALREADY_REGISTERED: 'Esiste già una seconda frase segreta registrata per questo indirizzo',
+    INSTRUCTIONS: 'Questa è la tua seconda frase segreta. Assicurati di fare un backup e di tenerla al sicuro!'
+  },
+
+  WALLET_SIDEBAR: {
+    FILTER: 'Filtro',
+    HIDE: 'Nascondi',
+    EXPAND: 'Espandi',
+    LOADING_LEDGER: 'Loading Ledger...',

```

Comment:
Is this correct?

Reply:
LOADING_LEDGER: 'Caricamento Ledger in corso...'